apps/fabric-website/src/pages/Styles/ElevationPage/docs/web/ElevationImplementation.md

```diff
@@ -1,35 +1,53 @@
 ### Fluent UI React (JavaScript variables)
 
+There are multiple ways to reference the depth variables in JavaScript.
+
+The recommended way is via the theme:
+
 ```jsx
-import { Depths } from '@uifabric/fluent-theme/lib/fluent/FluentDepths';
+import { getTheme } from '@fluentui/react';
 
-<div style={{ boxShadow: Depths.depth8 }} />;
+const theme = getTheme();
```

Comment:
this gets the global theme, right? Might need to add that caviat vs contextual theme.

Reply:
I thought about that, but I couldn't find a good existing explanation for that on the website and didn't want to spend time trying to write one (because it would take awhile)... I guess I could add a purposefully vague comment like "or other method to get the theme" to avoid having to explain it?

Reply:
this is where @xugao's useTheme hook might come in.

Reply:
we don't have a `useTheme` for v7 but do have one for v8 (in `react-theme-provider`).  
I would say leave this way and I will update this when post ship V8